concept-anchor.md

```diff
@@ -0,0 +1,45 @@
+## Title
+Concept Anchor for discussions with Management
+
+## Patlet
+Those wanting to bring Inner Source to organizations have difficulty starting a discussion with management. This can sometimes be made easier by anchoring Inner Source to more well known concepts such as Agile / DevOps / Lean.
+
+## Problem
+Often Inner Source is a term that's novel to organizations, making it often difficult to start a conversation with management about adopting Inner Source.
+
+## Context
+An individual sees an opportunity for Inner Source to make an impact within an organization. They would like to discuss Inner Source with management. Being a relatively traditional and bureaucratic company, it is difficult to broach the subject of Inner Source with management as they are reluctant to engage on a topic that they've neither heard of nor know anything about.
+
+These organizations might have an existing strategy, such as Agile, Lean, or DevOps transformation. To gain mindshare, it might be useful to start with these existing, well known concepts and build Inner Source on top of these concepts.
+
+## Forces
+What makes the problem difficult? What are the trade-offs? These are constraints that **can be changed** at a cost. The solution might change one or more of these forces in order to solve the problem, while also in-turn changing the context.
+
+## Sketch
+**TBD**
+
+## Solutions
+**TBD**
+
+## Resulting Context
+**TBD**
+
+## Rationale (optional)
+**TBD**
+
+## Known Instances (optional)
+**TBD**
+
+## Status (optional until merging)
+General pattern status is stored in GitHub's Label tagging - see any pull request. Note that this GitHub label tagging becomes less visible once the pattern is finalized and merged, so having some information in this field is helpful. 
+
+You might store other related info here, such as review history: "Three of us reviewed this on 2/5/17 and it needs John's expertise before it can go further."
+
+## Author(s)
+* Mishari A. Muqbil (Zymple)
+
+## Acknowledgements
+* Fei Wan
+* Sebastien Spier
+* Danes Cooper
```

Comment:
Danese

Reply:
And Sebasti**a**n :) Not sure how I qualified for the honours but that's my name :)